ecommerce/models.py

```diff
@@ -35,6 +35,7 @@ class Order(Model):
         choices=[(status, status) for status in STATUSES],
         default=CREATED,
         max_length=30,
+        db_index=True,
```

Comment:
I'm not sure we need this index, this column has low cardinality so the index may not give us much: http://blog.celerity.com/how-to-design-sql-indexes